src/pages/Home.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <main>
     <header>
-      <div class="logo"></div>
+      <img class="logo" src="@images/logo.png" alt="leveldb-inspector logo" />
     </header>
     <form @submit.prevent="connect">
       <p>Select path to LevelDB</p>
-      <label class="file db-path">
-        <input type="text" id="file" webkitdirectory />
-        <span class="file-custom" @click="selectPath">{{ path }}</span>
-      </label>
+      <div class="file" tabindex="0" @click="selectPath" @keyup.enter="selectPath">
+        <input type="text" id="file" :value="path" tabindex="-1" readonly />
```

Comment:
This PR seems to have the same changes as https://github.com/uncaughtxcptn/leveldb-inspector/pull/13 . Please check if we can close the other 1 or cleanup the commits in this PR

Reply:
@emanlodovice Not the same commits. This branch is based on #13 , but I used `master` as the target branch for the PR. Changed it to `bugfix/homepage-overflow` now for convenience.